src/OpenTelemetry.Exporter.OpenTelemetryProtocol/OtlpExporterOptions.cs

```diff
@@ -48,16 +48,10 @@ public class OtlpExporterOptions : IOtlpExporterOptions
     /// Initializes a new instance of the <see cref="OtlpExporterOptions"/> class.
     /// </summary>
     public OtlpExporterOptions()
-        : this(OtlpExporterOptionsConfigurationType.Default)
-    {
-    }
-
-    internal OtlpExporterOptions(
-        OtlpExporterOptionsConfigurationType configurationType)
         : this(
-              configuration: new ConfigurationBuilder().AddEnvironmentVariables().Build(),
-              configurationType,
-              defaultBatchOptions: new())
+            configuration: new ConfigurationBuilder().Build(),
```

Comment:
This is redundant - this now just creates an empty configuration. It could be refactored to avoid the need to do this.

Reply:
Changed to `IConfiguration?` and passed null here